cocos2d/core/platform/CCSys.js

```diff
@@ -551,6 +551,8 @@ else {
             browserType = sys.BROWSER_TYPE_MAXTHON;
         else if (browserType === "opr")
             browserType = sys.BROWSER_TYPE_OPERA;
+        else if (browserType === "chrome")
+            browserType = sys.BROWSER_TYPE_CHROME;
```

Comment:
There is no need to do this, the `typeReg2.exec(ua)` will assign directly `chrome` to `browserType`